### PR TITLE
Merge two ladspa test files

### PR DESCRIPTION
**Pull request by madwort**
* another typo!

### CODE REVIEW

**Review by sandreae — COMMENTED** [review]
Looks good, however I came across this on the ladspa_client PR https://github.com/noiseorchestra/jacktrip_pypatcher/pull/45 and thought this file was an accidental duplicate of `ltest_adspa_plugins` so removed it here https://github.com/noiseorchestra/jacktrip_pypatcher/pull/42/commits/6003aa249c393c01c0a35c6079aba90be80c9dd3

Is there a good order to merge these two PR in?

**Review by sandreae — DISMISSED** [review]
Probably best to merge this first as the other PR is a bit more in-depth :+1:

**Comment by madwort** [comment]
yep - I think the other PR should only have changes to tests that are directly to do with the new code - any tidying up / moving tests should be done in a seperate PR?

**Comment by madwort** [comment]
but actually - which of these two names is correct?

**Comment by madwort** [comment]
the other file is `ladspa_plugins.py` so I think this should be `test_ladspa_plugins.py` -> I'll update the PR when I get a min

**Comment by sandreae** [comment]
> the other file is `ladspa_plugins.py` so I think this should be `test_ladspa_plugins.py` -> I'll update the PR when I get a min

Yeh, I agree. Actually, I don't know where the other `ladspa_plugin.py` file came from, I thought it was a mistake when I saw it....

**Comment by madwort** [comment]
ok, looks good now?

**Review by sandreae — CHANGES_REQUESTED** [review]
I think this section of code is an error, right?

**Comment by madwort** [comment]
which one sorry?

File: test_ladspa_plugins.py [hunk 1]
```diff
@@ -16,4 +114,9 @@ def test_generate_subprocess_cmd(position=0.3):
         0.75,
     ]
 
-    assert ladspa.get_panning_positions(2, panning_positions) == [-0.45, -0.46, 0.45, 0.46]
+    assert ladspa.get_panning_positions(2, panning_positions) == [
+        -0.45,
+        -0.46,
+        0.45,
+        0.46,
+    ]
```
Comment (sandreae) [inline test_ladspa_plugins.py:122]: This whole function doesn't make sense, right?

**Review by sandreae — COMMENTED** [review]
> which one sorry?

Sorry, see below.... (the last function in `test_ladspa_plugins.py`

File: test_ladspa_plugins.py (hunk 1 above)
Comment (madwort) [reply test_ladspa_plugins.py:122]: ah, this is from before `ladspa` got converted to an object, so it's got the wrong function signature. Will revise this PR for the current codebase.

**Comment by madwort** [comment]
will re-do in the new context

**Pull request closed without merging**